engine/src/flutter/lib/ui/semantics.dart

```diff
@@ -362,6 +362,72 @@ enum SemanticsRole {
 
   /// The main display for a tab.
   tabPanel,
+
+  /// An input field for users to enter search terms.
+  searchBox,
+
+  /// A control that used for dragging across content.
+  ///
+  /// e.g, the drag handle of [ReorderableList].
```

Comment:
nit: here and elsewhere either capitalize "e.g." to "E.g.", or even better use the full "For example,"